packages/ddp-server/livedata_server.js

```diff
@@ -1352,8 +1382,10 @@ Object.assign(Subscription.prototype, {
       }
       ids.add(id);
     }
-
-    this._session.added(this._subscriptionHandle, collectionName, id, fields);
+    this._runWithThrottling(
+       () => this._session.added(this._subscriptionHandle, collectionName, id, fields)
+    )
+    this._lastUpdate = Date.now();
```

Comment:
1. As every `_runWithThrottling` call site sets `_lastUpdate`, it could happen in this function.
2. What if the subscription stopped before the `setTimeout` is triggered?

Reply:
> What if the subscription stopped before the setTimeout is triggered?

Not thought about this case.

Maybe moving the ``_lastUpdate`` statement to look like this:

```js
    this._runWithThrottling(
       () => { 
            this._session.added(this._subscriptionHandle, collectionName, id, fields);
            this._lastUpdate = Date.now();
      }
    )

```